src/components/shortcut_guide/components/NativeShortcutGuide.tsx

```diff
@@ -0,0 +1,52 @@
+/**
+ * This program is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General Public License as published by the Free Software Foundation, version 3 of the License.
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+ * You should have received a copy of the GNU Affero General Public License along with this program. If not, see <https://www.gnu.org/licenses/>.
+ * Other licensing options may be available, please reach out to data-viz@feedzai.com for more information.
+ */
+
+import { useRef, useEffect } from "react";
+import "../../../assets/style/NativeShortcutGuide.css";
+
+import * as constants from "../../../constants";
+import { ShortcutGuideDescription } from "./NativeGuideDescription";
+import { ShortcutGuideHeader } from "./NativeGuideHeader";
+import { ShortcutGuideBody } from "./NativeShortcutGuideBody";
+
+interface NativeShortcutGuideProps {
+	dialogRef: React.RefObject<HTMLDialogElement>;
+}
+
+/**
+ * Component that renders the list of all AutoVizuA11y shortcuts.
+ *
+ * @return Shortcut guide.
+ */
+const NativeShortcutGuide = ({ dialogRef }: NativeShortcutGuideProps): JSX.Element => {
+	const refNav = useRef(null);
+
+	useEffect(() => {
+		// Needs a slight delay since some elements take time to load
+		setTimeout(() => {}, 500);
```

Comment:
don't forget to clean up this setTimeout

Reply:
removed!